packages/app/src/hooks/useMetadata.tsx

```diff
@@ -15,6 +16,8 @@ import {
 import { getMetadata } from '@/metadata';
 import { toArray } from '@/utils';
 
+export const MAX_ROWS_TO_READ = DEFAULT_SAMPLE_SIZE;
```

Comment:
It seems like `MAX_ROWS_TO_READ` and `DEFAULT_SAMPLE_SIZE` are used interchangeably. Maybe we should just set it to a common name?